### DISCUSSION TITLE
How to change root from http://localhost/groupoffice to http://localhost ?

### DISCUSSION THREAD

**Discussion by tmikaeld** [Intermesh/groupoffice#1101] [Q&A] [answered]
This was possible in the previous versions, but it's now removed from docs.

How to do it in six.eight?

**Comment by mschering** [comment, marked as answer]
You have to modify the apache config. Set the DocumentRoot of your virtualhost to /usr/share/groupoffice.

**Comment by tmikaeld** [comment]
Thank you, that worked just fine!
